lib/1.4/dml-builtins.dml

```diff
@@ -492,6 +515,8 @@ The `device` template contains the following parameters:
 */
 template device {
     is object;
+    // _model_init is automatically called on model load
```

Comment:
move comment to `_model_init` definition?

Reply:
I commented this here for symmetry with the comments re. `_init` and `_post_init` immediately below. Although you could argue they should be moved as well.